### DISCUSSION TITLE
Config.ini location?

### DISCUSSION THREAD

**Discussion by optio50** [Louisvdw/dbus-serialbattery#933] [General]
What is the preferred location of this file?
it seems there have been two paths.
/data/etc/dbus-serialbattery/config.ini
/opt/victronenergy/dbus-serialbattery/config.ini

which is correct and will it survive a firmware upgrade?

**Comment by mr-manuel** [comment]
`/data/etc/dbus-serialbattery/config.ini` is the correct one.
`/opt/victronenergy/dbus-serialbattery/config.ini` gets overwritten by `/data/etc/dbus-serialbattery/config.ini` if you launch `reinstall-local.sh`.

Yes, `/data/etc/dbus-serialbattery/config.ini` survives firmware and driver upgrades.
